baud_rate: 3000000
device_name: "/dev/serial/by-id/usb-FTDI_USB__-__Serial_Converter_FT3R45P2-if00-port0"
start_pos: [0.0, 0.0]
min_position_deg: [-100.0, -100.0]
max_position_deg: [100.0, 100.0]
kP: 500
kI: 0
kD: 200
dynamixel_type: "XL430-W250"
